app/models/order.rb

```diff
@@ -0,0 +1,6 @@
+class Order < ApplicationRecord
+  belongs_to :user
```

Comment:
may be we can do something like this:
```suggestion
  belongs_to :buyer, class_name: "User", foreign_key: "user_id"
```

Reply:
Fixed it bhai. Thank you.